.github/workflows/ci.yml

```diff
@@ -126,6 +126,7 @@ jobs:
       - name: Run tests
         shell: bash
         run: |
+          export CONSUL_AGENT_CONFIG_PATH=$GITHUB_WORKSPACE/Consul.Test/test_config.json
```

Comment:
Instead of using export, you can use `env` context in line 133

Reply:
I switched it now 😄